src/app/core/utility-module/blogs-component.service.ts

```diff
@@ -11,13 +11,13 @@ export class BlogsComponentService {
 
   recentBlog: Blog; // holds the latest blogpost
 
-  constructor(private blogsService: BlogsService) {
-    this.blogsService.getMostRecentBlog().subscribe(
```

Comment:
Just confirming getMostREcentBlog and getAllBlogs are the same?

Reply:
Left a comment, `CLARK-SERVICE needs to accept a body with a get route, which currently it does not`, 
so getting the most recent blogs is unavailable for now.